gov2/kms/README.md

```diff
@@ -0,0 +1,96 @@
+# AWS SDK for Go V2 code examples for AWS AWS KMS 
+
+## Purpose
+
+These examples demonstrates how to perform several AWS Key Management Service (AWS KMS) 
+operations using version 2 of the AWS SDK for Go.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
```

Comment:
Just double-checking you want a link to v1 because you mention v2 earlier (but I see there's no v2 guide published yet).

Reply:
Good catch. Once we have the v2 guide available (hopefully by the end of next week), I'll have to go back and update all of these links.